automatic/vscode-insiders.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,38 @@
+$ErrorActionPreference = 'Stop'
+
+# Close Code Insiders before installing
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\chocolateyBeforeModify.ps1
```

Comment:
Do not source other chocolatey related scripts.
`chocolateyBeforeModify.ps1`, `chocolateyUninstall.ps1` and `chocolateyInstall.ps1` should never be sourced from other scripts, and will be requested to be changed by a moderator.

Reply:
Makes sense.

Reply:
You can rather move the code to a function inside a `helpers.ps1` file (convention in this repo), have that file sourced and called in the uninstall and install script (and remove the before modify script).

Remember additional error handling may be necessary in that case, to ensure the install/uninstall won't fail if a process can't be found.

Reply:
It won't be necessary to have additional error handling because the two `Get-Process` calls already have `-ea 0`.

Reply:
Ok, got it, you're saying that the code will run twice in the `update`?

Reply:
How should I create the `chocolateyUninstall.ps1` just for this call? I'm not sure if creating it will prevent the auto uninstall to run.

It's not exactly necessary, since double checking, if the program is open, won't hurt.

Reply:
Yup, indeed.